docs/storage/petalibrary/allocation_types.md

```diff
@@ -0,0 +1,38 @@
+## PetaLibrary Allocation Types
+
+#### Active
+
+PetaLibrary active allocations are the most performant PetaLibrary tier (please note that performant is a relative term -- our parallel scratch filesystem will outperform any PetaLibrary tier). The underlying hardware is configured to be suitable for direct compute access, and as such is accessible from all compute environments (Alpine, Blanca, Open OnDemand). All active allocations offer the following options:
+
+ - interactive (NFS) access from RC login and compute nodes
+ - accessible using Globus/rsync/sftp via the data transfer node (DTN) hosts
+ - SMB access (not enabled by default, only accessible from UCB campus network)
+ - no object (file/directory) limit
+ - double parity (up to two simultaneous disk failures will not cause data loss)
+ - snapshots (see documentation on [snapshots](ZFS_snapshots.md) for more information)
+ - full data checksumming, validated monthly (via ZFS scrub)
+ - lz4 data compression
+
+#### Archive
+
+PetaLibrary archive allocations are configured to maximize data integrity over performance. As such, archive allocations are not accessible from compute nodes, but can be accessed interactively on login nodes. Historically the archive tier was tape-based, although recently is was changed to disk. An object limit of 10,000 files/directories per terabyte of space remains in place to preserve the option to move back to tape in the future, should the service grow to a point where tape-based storage is more fiscally viable.
+
+The archive feature list is similar to active, with the major differences being: no direct access from RC compute, no SMB access, file/directory count is limited, triple parity is used for data protection rather than double parity, and a different compression algorithm is used. In summary, all archive allocations offer the following features:
+
+ - interactive (NFS) access from login nodes
+ - accessible using Globus/rsync/sftp via the data transfer node (DTN) hosts
+ - 10,000 object (file/directory) per terabyte limit
+ - triple parity (up to three simultaneous disk failures will not cause data loss)
+ - snapshots (see documentation on [snapshots](ZFS_snapshots.md) for more information)
```

Comment:
Restating: 

This link is broken, I think you just need to put the correct name, I believe ZFS is lowercase. Please correct it and then ensure that it works using the checks in the PR that generate the documentation.